content/3_Silver/Conclusion.problems.json

```diff
@@ -202,6 +202,19 @@
         "site": "CF"
       }
     },
+    {
+      "uniqueId": "cf-404C",
+      "name": "Restore Graph",
+      "url": "https://codeforces.com/contest/404/problem/C",
+      "source": "CF",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Graphs", "DFS", "BFS"],
```

Comment:
```suggestion
      "tags": ["BFS"],
```

Reply:
does dfs not work for this problem or smth

Reply:
idk